src/main_logic/main.cpp

```diff
@@ -49,22 +49,26 @@ int main()
                                             (RunIterationFunc)wrapped_run_iteration_process_particles,
                                             wrapped_run_iteration_cleanup};
 
+    std::pair<std::string, std::string> visualization_endpoints = get_visualization_endpoints();
 
-    std::string visualization_endpoint = get_visualization_endpoint();
-
-
-    while(true)
+    bool modelDispatchFail = false;
```

Comment:
I would call it `modelDispatchFailed` because the current name sounds like a prompting (i. e. go fail) :|